slides/snippets/src/main/java/CopyPresentation.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// [START slides_copy_presentation]
+
+import com.google.api.client.googleapis.auth.oauth2.GoogleCredential;
+import com.google.api.client.googleapis.javanet.GoogleNetHttpTransport;
+import com.google.api.client.json.jackson2.JacksonFactory;
+import com.google.api.services.drive.Drive;
+import com.google.api.services.drive.model.File;
+import com.google.api.services.slides.v1.SlidesScopes;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class CopyPresentation {
+
+    // Create a new presentation
+    public static void copyPresentation(String sourcePresentationId) {
+        try {
+            // String sourcePresentationId = "55BYRgExa5NEv9vMegpPBaWESy7oKTK21l4P2DcX5l4";
```

Comment:
What's the idea behind including these commented out lines? Should there be an instruction explaining them? If they work out of the box, should the variable type declaration be removed, since it would result in an error?

Reply:
Same premise as what we've already been doing with declaring them then overwriting the values in excluded regions -- want to bias towards using/showing concrete values as much as possible. This inverts it so we're not overwriting the value, but using the comment to show the concrete values.

Reply:
I worry that these comments will be more likely to fall out of date, during refactors, etc. What about having a main method in each sample which does a call to it? Something like:

```
public static void main(String[] args) {
  String sourcePresentationId = "55BYRgExa5NEv9vMegpPBaWESy7oKTK21l4P2DcX5l4";
  copyPresentation(sourcePresentationId)
}
```